modules/oe_translation_cdt/config/install/field.storage.oe_translation_request.contact_usernames.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - oe_translation
+id: oe_translation_request.contact_usernames
+field_name: contact_usernames
```

Comment:
I named it after CDT's property name `contact_user_names` (reference: https://github.com/openeuropa/cdt-client/blob/72e9de7752ebeda0872526c4176681dfcc524c6d/src/Model/Request/Translation.php#L21). After the Wednesday call we agreed that the CDT and ePoetry won't be installed together, but it would be good to avoid naming conflicts anyway.